### DISCUSSION TITLE
what is the difference between ubl-bltouch and just bltouch

### DISCUSSION THREAD

**Discussion by blackoutlaw** [Jyers/Marlin#1066] [General Discussion]
i just noticed that there are two different compiled firmware one that is just bltouch and then one that is ubl bltouch.  so what is the difference and how does the ubl work

**Comment by smooreace** [comment]
One has unified bed leveling and the other doesnt

**Comment by blackoutlaw** [comment]
Do what is unified bed leveling?

Sent from the all new AOL app for Android 
 
  On Thu, Jun 24, 2021 at 21:50, ***@***.***> wrote:   

One has unified bed leveling and the other doesnt

**Reply by smooreace** [reply]
https://www.google.com/search?q=unified+bed+leveling&oq=unified+bed+le&aqs=chrome.0.0l2j69i57j0l3j69i60j69i61.5699j1j7&sourceid=chrome&ie=UTF-8
